core/storage/collection/gae_models.py

```diff
@@ -162,7 +162,7 @@ class CollectionRightsModel(base_models.VersionedModel):
             constants.ACTIVITY_STATUS_PUBLIC
         ]
     )
-    # DEPRECATED in v2.8.1. Do not use.
+    # DEPRECATED in v2.8.2. Do not use.
```

Comment:
Here and elsewhere, 2.8.2 seems wrong. 2.8.2 is already out, see https://github.com/oppia/oppia/releases

I think you mean 2.8.3.

Reply:
Done.